pinot-spi/src/main/java/org/apache/pinot/spi/config/table/QueryConfig.java

```diff
@@ -37,14 +37,27 @@
   // because by the time the server times out, the broker should already timed out and returned the response.
   private final Long _timeoutMs;
 
+  // Table config override for disable Groovy broker property
+  private final Boolean _disableQueryGroovy;
+
   @JsonCreator
-  public QueryConfig(@JsonProperty("timeoutMs") @Nullable Long timeoutMs) {
+  public QueryConfig(@JsonProperty("timeoutMs") @Nullable Long timeoutMs,
+      @JsonProperty("disableQueryGroovy") @Nullable Boolean disableQueryGroovy) {
```

Comment:
Since this is query config, I think we can simplify the config key to `disableGroovy`?